avm/res/synapse/workspace/main.bicep

```diff
@@ -66,6 +66,9 @@ param preventDataExfiltration bool = false
 @description('Optional. Enable or Disable public network access to workspace.')
 param publicNetworkAccess string = 'Enabled'
 
+@description('Optional. List of firewall rules to be created in the workspace.')
+param firewallRules array = []
```

Comment:
I'd propose changing this to 
```bicep
param firewallRules firewallRuleType[]?
```
and add a corresponding type
```bicep
type firewallRuleType = {
  @description('Required. The name of the firewall rule.')
  name: string
  
  @description('Required. The start IP address of the firewall rule. Must be IPv4 format.')
  startIpAddress: string
  
  @description('Required. The end IP address of the firewall rule. Must be IPv4 format. Must be greater than or equal to startIpAddress.')
  endIpAddress: string
}
```

And adjusting the usage of the type to
```bicep
module workspace_firewallRules 'firewall-rules/main.bicep' = [
  for (rule, index) in (firewallRules ?? []): {
```
to not have it complain about the nullable parameter 😉 

Once done, please refresh the readme & json

Reply:
Hi @AlexanderSehr i've applied all the changes and action status is green